### PR TITLE
#155738443 Create layout file for RecyclerView

### PR DESCRIPTION
**Pull request by codestaintin**
#### What does this PR do?
- Create layout file for RecyclerView

#### Description of Task to be completed?
- imported recycleView widget to gradle file
- add class to get user details
- create layout file for RecyclerView

#### How should this be manually tested?
N/A
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?

- story type: feature
- story id: 155738443
- story title: Create layout file for RecyclerView

#### Screenshots (if appropriate)

N/A
#### Questions

N/A

[Finishes #155738443]

### CODE REVIEW

File: app/src/main/java/com/example/isioyemohammed/gitlist/ListItem.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package com.example.isioyemohammed.gitlist;
```
Comment (TheDancerCodes) [inline app/src/main/java/com/example/isioyemohammed/gitlist/ListItem.java:1]: @codestaintin 
Should this file be part of this PR. 

The PR title is *Create layout file for RecyclerView*

**Review by TheDancerCodes — APPROVED** [review]
Besides the model file **ListItem.java** that I don't think should have been in this PR, every thing else LGTM

File: app/src/main/java/com/example/isioyemohammed/gitlist/ListItem.java (hunk 1 above)
Comment (codestaintin) [reply app/src/main/java/com/example/isioyemohammed/gitlist/ListItem.java:1]: @TheDancerCodes , thanks for the feedback, duly noted! I thought I was on a separate branch when I created the ListItem.java file, will pay more attention next time.

Comment (TheDancerCodes) [reply app/src/main/java/com/example/isioyemohammed/gitlist/ListItem.java:1]: Great! 👍 @codestaintin

**Pull request merged**